localization/react-intl/src/app/components/media/MediaStatusCommon.json

```diff
@@ -1,7 +1,6 @@
 [
   {
     "id": "mediaStatus.error",
-    "description": "Error message displayed when a status change fails",
```

Comment:
why was this description removed?

Reply:
i don't know. I did not do that intentionally. Either it was removed some other way or a typo. I will re-add it.

Reply:
put it back